README.md

```diff
@@ -0,0 +1,78 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce 3D data visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista offers 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to create 3D visualizations. 
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same as Abstract.
+
+### Tutorial Outline.
+
+0. Getting Started - Why PyVista? (10 min for talk, 10 min for exercise)
+
+1. Basic usage - Reading and plotting 3D data.  (10 min for talk, 10 min for exercise)
+
+2. What is a Mesh? - Learn about "Point", "Cell" and "Attributes". (10 min for talk, 10 min for exercise)
+
+3. PyVista Data Model - Learn about the data model of the PyVista framework. (10 min for talk, 5 min for exercise)
+
+4. Mesh Creation - Exercise to build and plot meshes in PyVista. (10 min for talk, 5 min for exercise)
+
+6. Charts API - Exercise to use the Charts API to plot your data in PyVista. (10 min for talk, 5 min for exercise)
+
+7. Unique uses of PyVista -  About some unique applications of PyVista. (10 min for talk, 5 min for exercise)
+
+8. External Examples - How PyVista is used in external projects. (15 min for talk)
+
+### Additional Tutorial Information.
+
+We will base the material for the tutorial on [the examples in PyVista's online documentation](https://docs.pyvista.org/examples/index.html).
+The tutorial itself will be in the [pyvista-tutorial](https://github.com/pyvista/pyvista-tutorial) repository.
+
+### Tutorial Prerequisites.
+
+1. Basic knowledge of NumPy.
+
+
+
+2. It is ideal if you are a heavy user of VTK.
```

Comment:
I think users should be able to use PyVista without any knowledge of VTK, though it could be seen that PyVista is a way to be gently "brought into" VTK without the difficult API.

```suggestion
#. Basic knowledge of Python to get started. Get jupyterlab on your machine and be up and running.
#. Intermediate users will want to be familiar with [NumPy](https://numpy.org/) and other libraries that are compatible with PyVista, like `trimesh` or `meshio`.
```